src/rust-crypto/DehydratedDeviceManager.ts

```diff
@@ -304,3 +318,30 @@ export class DehydratedDeviceManager {
         }
     }
 }
+
+/**
+ * The events fired by the DehydratedDeviceManager
+ * @internal
+ */
+export type DehydratedDevicesEvents =
```

Comment:
does this need to be `exported`? It doesn't seem to be used elsewhere?

(Alternatively, maybe import it to `rust-crypto.ts` and use it to help with the re-emits)